runtime/src/accounts_db.rs

```diff
@@ -9617,13 +9616,13 @@ pub mod tests {
         }
     }
 
-    impl<'a> BankHashLamportsVerifyConfig<'a> {
+    impl<'a> VerifyAccountsHashAndLamportsConfig<'a> {
         fn new_for_test(
             ancestors: &'a Ancestors,
             epoch_schedule: &'a EpochSchedule,
             rent_collector: &'a RentCollector,
-        ) -> BankHashLamportsVerifyConfig<'a> {
-            BankHashLamportsVerifyConfig {
+        ) -> VerifyAccountsHashAndLamportsConfig<'a> {
+            VerifyAccountsHashAndLamportsConfig {
```

Comment:
I imagine you just used a refactoring tool, but we can probably just do:
```suggestion
        ) -> Self {
            Self {
```